### ISSUE TITLE
numberOfMonths: 1 but It shows 2 crushed months

### ISSUE THREAD

**Issue by ArthurBugan** [airbnb/react-dates#712]
Hello, I'm starting to implement react-dates on my site, but I already set numberOfMonths to 1 and this wierd thing ( on screenshot ) happens, If somebody could help me!

![imageedit_2_6289452914](https://user-images.githubusercontent.com/15329556/30382310-2b21cbb4-9875-11e7-95fc-f9bd9610eafe.png)

**Comment by majapw** [comment]
@ArthurBugan this usually happens because there is some extra CSS being applied universally to table elements. Can you inspect the element in chrome to see if that's the case?

This can happen if you're using react-dates in conjunction with something like materialcss or bootstrap.

**Comment by majapw** [comment]
Does this happen with a different numberOfMonths value?

**Comment by ArthurBugan** [comment]
@majapw Yes It happens with more months aswell, in this SS I put the numberOfMonths to 4, Its a CSS problem I think, I'm going to check the classes

![imageedit_4_8346551120](https://user-images.githubusercontent.com/15329556/30384926-fa87bf02-987b-11e7-99a2-93da102716ba.png)
